src/templates/_layouts/parts-kit.html

```diff
@@ -46,7 +59,8 @@ <h3>{{ title | title }}</h3>
                                 data-togglee
                             >
                                 {% for item in section.items %}
-                                    <li>
+                                    <li data-section-item
+                                        data-search-text="{{ title ~ ' ' ~ item.title }}">
```

Comment:
I kept the search text pretty dumb here. It just combines the section title with the individual items title. 

The idea is you could search "button" and see all of the buttons (if they were in a group like that). Or you could search primary and it would only show a component named "button primary".

Reply:
makes sense 👍